_authors/osci_2023/strifale.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Sharvil Trifale
+short_name: Sharvil
+github: strifale
+linkedin: linkedin.com/in/strifale
```

Comment:
Hi @strifale , please only include the linkedin username and not a full URL

Reply:
Otherwise this LGTM!

Reply:
@cwperks  updated!